deepmd/fit/ener.py

```diff
@@ -14,6 +14,9 @@
 from deepmd.env import global_cvt_2_tf_float
 from deepmd.env import GLOBAL_TF_FLOAT_PRECISION, TF_VERSION
 
+from deepmd.nvnmd.utils.config import nvnmd_cfg
+from deepmd.nvnmd.fit.ener import one_layer_nvnmd, one_layer_deepmd
```

Comment:
`one_layer_deepmd` is actually the `one_layer` provided by `deepmd.utils.network`. 
I suggest only import `one_layer_nvnmd` or revise it as
```
from deepmd.utils.network import  one_layer as one_layer_deepmd
```